pkgs/development/compilers/paco/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "paco";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "pacolang";
+    repo = "paco";
+    rev = "5d5c7d3c13558afa6428b5ee2c7a9b80c69e56de";
```

```suggestion
    rev = "v${version}";
```

When the project uses tags to mark releases, we prefer to use them as `rev` attribute here, instead of raw MD5 git commit strings.